README.md

```diff
@@ -265,6 +267,26 @@ There are following known limitations of native OTel logs collection:
   Logs collection with fluentd is still recommended if chart deployed with `autodetect.istio=true`.
 - Journald logs cannot be collected natively by Splunk OTel Collector yet.
 
+### Performance of native OpenTelemetry logs collection
+
+Some configurations used with Splunk Connect for Kubernetes-OpenTelemetry can have an impact on overall performance of log ingestion. The more receivers, processors, exporters and extensions that are added to any of the pipelines, the greater the performance impact.
```

Comment:
Suggested change: " ... processors, exporters, and extensions that are ..."

Reply:
Hi @tmorrisonsfx , thank you for all the suggestions! Can you elaborate on this one more?

Reply:
Yes, it's just to add a comma after exporters. Sorry for the confusion. Thanks!